src/components/TabsButtons/TabsButtons.tsx

```diff
@@ -14,15 +14,18 @@ export const TabsButtons: React.FC<Props> = ({ tabs, selectedTab, onClick }) =>
       tabs.map(tab => (
         <button
           key={tab.id}
-          id={tab.id}
           className={classNames(
             'tabs__button',
             {
               'tabs__button--active': tab.id === selectedTab,
             },
           )}
           type="button"
-          onClick={() => onClick(tab.title, tab.id, tab.content)}
+          onClick={
+            tab.id !== selectedTab
+              ? () => onClick(tab.title, tab.id, tab.content)
+              : () => 0
```

Comment:
Why did you decide to use ternary? There better use if condition or `and` logical operator